test/block.ts

```diff
@@ -168,73 +192,23 @@ function initBlockTestCase() {
     state.Nop();
   }
 
-  let block = state.forge();
-  return block;
+  let blocks = state.forgeAllL2Blocks();
+  return blocks;
 }
 
 let block_test_case = initBlockTestCase();
-class TestBlock implements SimpleTest {
-  getInput() {
-    let input = {
-      txsType: block_test_case.txsType,
-      encodedTxs: block_test_case.encodedTxs,
-      balance_path_elements: block_test_case.balance_path_elements,
-      order_path_elements: block_test_case.order_path_elements,
-      account_path_elements: block_test_case.account_path_elements,
-      orderRoots: block_test_case.orderRoots,
-      oldAccountRoots: block_test_case.oldAccountRoots,
-      newAccountRoots: block_test_case.newAccountRoots,
-    };
-    //console.log(JSON.stringify(input, null, 2));
-    return input;
-  }
-  getOutput() {
-    return {};
-  }
-  getComponent(): TestComponent {
-    return {
-      src: path.join(__dirname, '..', 'src', 'block.circom'),
-      main: `Block(${nTxs}, ${balanceLevels}, ${orderLevels}, ${accountLevels})`,
-    };
-  }
-}
 
-function initEmptyBlockTestCase() {
+function initEmptyBlockTestCase(): common.L2Block {
   let state = new GlobalState(balanceLevels, orderLevels, accountLevels, nTxs);
   // we need to have at least 1 account
   state.createNewAccount();
-  for (var i = state.bufferedTxs.length; i < nTxs; i++) {
+  for (var i = 0; i < nTxs; i++) {
     state.Nop();
   }
```

Comment:
maybe we can remove this because now we have
```
  forge(autoPad = true): L2Block {
    if (autoPad) {
      this.flushWithNop();
    }
    return this.forgeWithTxs(this.bufferedTxs);
  }
  flushWithNop() {
    while (this.bufferedTxs.length % this.nTx != 0) {
      this.Nop();
    }
  }
  forgeAllL2Blocks(): Array<L2Block> {
    this.flushWithNop();
    return this.bufferedBlocks;
  }
```

Reply:
Both is ok.